examples/plugins/src/knip/README.md

```diff
@@ -0,0 +1,89 @@
+# knip
+
+🕵️ **Code PushUp plugin for detecting unused code** 📋
+
+---
+
+## Getting started
+
+1. If you haven't already, install [@code-pushup/cli](../cli/README.md) and create a configuration file.
+
+2. Copy the [plugin source](../file-size) as is into your project
+
+3. Add this plugin to the `plugins` array in your Code PushUp CLI config file (e.g. `code-pushup.config.js`).
+
+   Pass in the path on the directory to crawl (relative to `process.cwd()`), as well as patterns and a budget.
+
+   ```js
+   import knipPlugin from './knip.plugin';
+
+   export default {
+     // ...
+     plugins: [
+       // ...
+       knipPlugin({}),
+     ],
+   };
+   ```
+
+See a detailed guide on how to configure knip on their [official docs](https://knip.dev/guides/handling-issues)
+
+4. (Optional) Reference audits (or groups) that you wish to include in custom categories (use `npx code-pushup print-config` to list audits and groups).
+
+   Assign weights based on what influence each audit and group should have on the overall category score (assign weight 0 to only include it for extra info, without influencing the category score).
+
+   ```js
+   import fileSizePlugin, { recommendedRefs as fileSizeRecommendedRefs } from './file-size.plugin';
+
+   export default {
+     // ...
+     categories: [
+       // ...
+       {
+         slug: 'performance',
+         title: 'Performance',
+         refs: [...fileSizeRecommendedRefs],
+       },
+     ],
+   };
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏Wrong plugin here